tests/unit/style/properties/serialization.rs

```diff
@@ -1231,4 +1231,26 @@ mod shorthand_serialization {
             assert_eq!(serialization, block_text);
         }
     }
+      mod effects {
+        pub use super::*;
+        pub use style::properties::longhands::box_shadow::SpecifiedValue as BoxShadow;
+        pub use style::values::specified::Shadow;
+        #[test]
+        fn box_shadow_should_serialize_correctly() {
+            let mut properties = Vec::new();
+            let color = Some(CSSColor {
```

Comment:
Let's move the `property_declaration_block` function to the parent module, and write this test as follows, which will be easier to maintain:

```rust
let shadow_css = "box-shadow: 1px 2px 3px 4px";
let shadow = property_declaration_block(shadow_css);
assert_eq!(shadow.to_css_string(), shadow_css);
```